src/main/java/org/osc/controller/nsc/api/SampleSdnRedirectionApi.java

```diff
@@ -324,14 +324,18 @@ public void deleteNetworkElement(NetworkElement portGroupId) throws Exception {
     public void removeInspectionPort(InspectionPortElement inspectionPort)
             throws NetworkPortNotFoundException, Exception {
         // no-op
+
+        if (inspectionPort == null) {
+            throw new IllegalArgumentException("Attempt to delete null inspection port!");
+        }
+
+        InspectionPortElement foundInspectionPort = getInspectionPort(inspectionPort);
```

Comment:
if inspection port is not found, return or throw exception(maybe NetworkPortNotFoundException) since there is nothing to be done.